examples/ai-core/src/generate-image/fal-kontext.ts

```diff
@@ -9,7 +9,7 @@ async function main() {
     prompt: 'Put a donut next to the flour.',
     providerOptions: {
       fal: {
-        image_url:
+        imageUrl:
           'https://v3.fal.media/files/rabbit/rmgBxhwGYb2d3pl3x9sKf_output.png',
       },
```

Comment:
Can you add `satisfies FalImageProviderOptions` for type checking. We recommend it as best practice so add it to examples now

```suggestion
fal: {
        imageUrl:
'https://v3.fal.media/files/rabbit/rmgBxhwGYb2d3pl3x9sKf_output.png',
} satisfies FalImageProviderOptions,
```